.vscode/settings.json

```diff
@@ -56,5 +56,6 @@
   ],
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
-  }
+},
+"python.languageServer": "Jedi"
```

Comment:
did you mean to change this?

Reply:
yeah VSCode is really intent on putting that in there